src/main/java/org/skriptlang/skript/bukkit/recipes/RecipeModule.java

```diff
@@ -0,0 +1,66 @@
+package org.skriptlang.skript.bukkit.recipes;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.ClassInfo;
+import ch.njol.skript.classes.EnumClassInfo;
+import ch.njol.skript.expressions.base.EventValueExpression;
+import ch.njol.skript.registrations.Classes;
+import ch.njol.skript.util.RecipeUtils;
+import org.bukkit.inventory.Recipe;
+import org.bukkit.inventory.recipe.CookingBookCategory;
+import org.bukkit.inventory.recipe.CraftingBookCategory;
+import org.skriptlang.skript.lang.comparator.Comparator;
+import org.skriptlang.skript.lang.comparator.Comparators;
+import org.skriptlang.skript.lang.comparator.Relation;
+
+import java.io.IOException;
+
+public class RecipeModule {
+
+	public static void load() throws IOException {
+		if (!Skript.classExists("org.bukkit.inventory.Recipe"))
+			return;
+
+		Skript.getAddonInstance().loadClasses("org.skriptlang.skript.bukkit.recipes", "elements");
+
+		Classes.registerClass(new ClassInfo<>(Recipe.class, "recipe")
+			.user("recipes?")
+			.name("Recipe")
+			.description("Represents a recipe")
+			.usage("recipes")
+			.examples("all recipes")
+			.since("INSERT VERSION")
+			.defaultExpression(new EventValueExpression<>(Recipe.class)));
+
+		Classes.registerClass(new EnumClassInfo<>(CraftingBookCategory.class, "craftingbookcategory", "crafting book categories")
+			.user("crafting ?book ?categor(y|ies)")
+			.name("Crafting Book Category")
+			.description("Represents the different categories of crafting recipes")
+			.since("INSERT VERSION")
+		);
+
+		Classes.registerClass(new EnumClassInfo<>(CookingBookCategory.class, "cookingbookcategory", "cooking book categories")
+			.user("cooking ?book ?categor(y|ies)")
+			.name("Cooking Book Category")
+			.description("Represents the different categories of cooking recipes")
+			.since("INSERT VERSION")
+		);
```

Comment:
Do we really need to separate these? I think it could be nicer for us to have a unified category type, which would remove the need for specifying cooking/crafting on the user's end.